lua/cfcfactions/core/cl/cl_clientstartup.lua

```diff
@@ -44,27 +44,35 @@ function cfcFactions:DisplayMenu()
 end
 
 --If the defined key is properly set, users can use that specific key to also open/close the derma
-hook.Add("PlayerButtonDown", "CFC_FAC_MENUKEYDOWN", function(ply, button)
+local function menuKeyDown(ply, button)
     if input.GetKeyName(button) == cfcFactions.Config.CLIENT_KEY then
         cfcFactions:DisplayMenu()
     end
-end)
+end
+
+hook.Add("PlayerButtonDown", "CFC_FAC_MENUKEYDOWN", menuKeyDown)
```

Comment:
This hook name should be CamelCase

```suggestion
hook.Add("PlayerButtonDown", "CFC_Fac_MenuKeyDown", menuKeyDown)
```